app/jobs/application_job.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationJob < ActiveJob::Base
+  def serialize
+    super.merge(foodcoop: FoodsoftConfig.scope)
+  end
+
+  def deserialize(job_data)
+    FoodsoftConfig.select_multifoodcoop job_data['foodcoop']
+    super(job_data)
+  end
```

Comment:
Will this also work when running inline (without a background-job backend)?

Reply:
since it runs inline there is no need to switch the database :wink:

Reply:
hah, you got me ;)